plugins/UsersManager/API.php

```diff
@@ -1297,18 +1297,16 @@ private function isUserTheOnlyUserHavingSuperUserAccess($userLogin)
      * If the username/password combination is incorrect an invalid token will be returned.
      *
      * @param string $userLogin Login or Email address
-     * @param string $md5Password hashed string of the password (using current hash function; MD5-named for historical reasons)
+     * @param string $passwordConfirmation the current user's password.
```

Comment:
Could we leave a comment: We recommend to always set the `passwordConfirmation` in a `$_POST` so the plain password won't appear in access logs. Ideally we would be able to force that it's sent over `$_POST` even? But of course this would only work if the root request is an API request for this method. We probably wouldn't be able to check `!empty($_POST['passwordConfirmation']) && empty($_GET['passwordConfirmation])` in other cases. Not sure if we can implement this check using `getRootApiRequestMethod()` but could be interesting to force people to POST the password and to force best practice here.

Reply:
I'll update the comment, but forcing `$_POST` seems non-trivial. Can we create an issue for it?

Reply:
I reckon it's not too important. Can simply put a comment there then. Although it might be trivial when checking if `Request::getRootApiRequestMethod() === 'UsersManager.create...'`. After all we also allow the token to be passed as GET so there's not really any difference and it's not needed. Comment is definitely enough.